two_factor/templates/two_factor/admin/setup.html

```diff
@@ -20,7 +20,7 @@
         smartphone to scan the QR code below. For example, use Google
         Authenticator. Then, enter the token generated by the app.
         {% endblocktrans %}</p>
-    <p><img class="centered-image" src="{{ QR_URL }}" alt="QR Code" /></p>
+    <p><img class="centered-image" style="background-color: white" src="{{ QR_URL }}" alt="QR Code" /></p>
```

Comment:
this needs to be fixed in the CSS, not with inline styles as Content-Security Policies will block this (inline styles are typically blocked)